rpcwebsocket.go

```diff
@@ -1774,7 +1775,8 @@ out:
 						if ok {
 							resp, err = wsHandler(c, cmd.params)
 						} else {
-							resp, err = c.rpcServer.standardCmdResult(cmd, nil)
+							resp, err = c.rpcServer.standardCmdResult(context.Background(),
```

Comment:
This should probably be a `context.TODO()` because this really should be coming from the overall WebSocket connection and passed into the `inHandler`.  However, since that infrastructure is not yet in place. this should be `context.TODO()` imo.